src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -38,9 +44,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <!-- All configurations -->
+    <Compile Include="System\Net\Sockets\SocketReceiveFromResult.cs" />
+    <Compile Include="System\Net\Sockets\SocketReceiveMessageFromResult.cs" />
+    <Compile Include="System\Net\Sockets\SocketTaskExtensions.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetGroup)' != 'net46'">
```

Comment:
Please add a comment here describing what SKU will use this code? What does it mean _not_ to be net46?